README.md

```diff
@@ -21,8 +21,10 @@ Then run it with:
     circleci local execute --job test
 
 ## Configuration
-By default, the service listens on port 8888. This is specified
-in `server.py` in the constructor for the `Server` class.
+All configuration is done using environment variables.
+To get the list of possible configuration parameters and their defaults, run
+
+    python -m file_catalog --show-config-spec
```

Comment:
This is appreciated. Being able to interrogate the program as to what it thinks its configuration is seems very useful for debugging.